tensorflow_addons/activations/rrelu_test.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from absl.testing import parameterized
+
+import math
+
+import numpy as np
+import tensorflow as tf
+from tensorflow_addons.activations import rrelu
+from tensorflow_addons.utils import test_utils
+import random
+
+
+def _ref_rrelu(x, alpha, training=None):
+    return tf.where(x >= 0, x, alpha * x)
+
+
+@test_utils.run_all_in_graph_and_eager_modes
+class RreluTest(tf.test.TestCase, parameterized.TestCase):
+    @parameterized.named_parameters(("float16", np.float16),
+                                    ("float32", np.float32),
+                                    ("float64", np.float64))
+    def test_rrelu(self, dtype):
+        x = tf.constant([-2.0, -1.0, 0.0, 1.0, 2.0], dtype=dtype)
+        lower = 0.2
+        upper = 0.2
+        result, alpha = rrelu(x, lower, upper, training=True)
```

Comment:
Do you have any  idea about random test? @WindQAQ

Reply:
I think we do not have to return `alpha` for rrelu OP. In your experiment, setting seed in TF is not enough, right?

Reply:
Argument with_alpha can control the return behavior and I don't need seed

Reply:
@fsx950223 I do not think it's a good practice to have an argument to return such things for public API. Is alpha deterministic while setting the same TF seed?

Reply:
No, it isn't and I don't know how to let alpha deterministic.

Reply:
Umm, I think we could just test values with `training=False`. As for gradient testing, we should check both of them. How do feel about this?

Reply:
changed